release-notes/opensearch-release-notes-2.0.0.md

```diff
@@ -0,0 +1,637 @@
+# OpenSearch and Dashboards 2.0.0 Release Notes
+
+## Release Highlights
+
+* Document level alerting allows users to create monitors that can generate alerts per document.
+* Lucene 9 is now used in OpenSearch
+* The Geo Map Tiles in OpenSearch Dashboards are updated and now have a pipeline to update them more frequently.
+* Document level security now supports term lookup queries.
+
+### OpenSearch Notifications
+* OpenSearch 2.0.0 is the first official release with OpenSearch Notifications.
+* Notifications consist of three plugins, `notifications-core` and `notifications` backend plugins for OpenSearch, and a `notificationsDashboards` frontend plugin for OpenSearch Dashboards.
+
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.0 include the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.0/release-notes/opensearch.release-notes-2.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.0/release-notes/opensearch-dashboards.release-notes-2.0.0.md).
+
+## FEATURES
+
+### OpenSearch Index Management
+* Adds shrink action to ISM ([#326](https://github.com/opensearch-project/index-management/pull/326))
+* Notification integration with IM ([#338](https://github.com/opensearch-project/index-management/pull/338))
+
+
+### OpenSearch Index Management Dashboards Plugin
+* Adds UI for shrink action ([#176](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/176))
+
+
+### OpenSearch Job Scheduler
```

Comment:
Is this a feature or an enhancement? It sounds like an enhancement, but I wasn't sure.

Reply:
@downsrob Could you help confirm this one?

Reply:
Upon further digging this looks like it is the right spot.